pkg/image/admission/imagequalify/api/v1/register.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// SchemeGroupVersion is group version used to register these objects
+var SchemeGroupVersion = schema.GroupVersion{Group: "", Version: "v1"}
```

Comment:
@deads2k are we putting new config types in groups now? `apiserver.config.openshift.io`? I couldn't find any other config types in origin yet

Reply:
> @deads2k are we putting new config types in groups now? apiserver.config.openshift.io? I couldn't find any other config types in origin yet

"webconsole.config.openshift.io" from here https://github.com/openshift/api/blob/master/webconsole/v1/register.go#L10

For admission plugins it's going to be a question of versioning cadence.  Because our admission plugins will be used in the openshift apiserver, I'm inclined to go "admission.config.openshift.io" without further subdivision.

Reply:
> For admission plugins it's going to be a question of versioning cadence. Because our admission plugins will be used in the openshift apiserver, I'm inclined to go "admission.config.openshift.io" without further subdivision.

Is there any change I need to make here?

Reply:
```
var GroupName = "admission.config.openshift.io"
var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1"}
```

wherever you have config data you load:
```
apiVersion: admission.config.openshift.io/v1
```